client/ayon_flame/plugins/publish/extract_product_resources.py

```diff
@@ -399,6 +399,12 @@ def process(self, instance):
                 representation_data))
 
             if export_type == "Sequence Publish":
+                publish_clips = flame.find_by_name(
+                    f"{exporting_clip.name.get_value()}_publish",
+                    parent=exporting_clip.parent
+                )
+                for publish_clip in publish_clips:
+                    flame.delete(publish_clip)
                 # at the end remove the duplicated clip
                 flame.delete(exporting_clip)
```

Comment:
wasn't this already removing the clip? Or is this doing anything at all, if not we should perhaps remove the code, right?